docs/developer/users.mdx

```diff
@@ -167,12 +184,84 @@ mutation {
     user {
       email
       isActive
+      isConfirmed
+    }
+  }
+}
+```
+
+If the token is valid, the user will be successfully confirmed:
```

Comment:
Maybe it would be worth to mention that the token has own TTL, that can be controled by settings.env(we can link the env here also)